src/overlays/actors/ovl_En_Ex_Ruppy/z_en_ex_ruppy.c

```diff
@@ -1,15 +1,28 @@
 #include "z_en_ex_ruppy.h"
-
+#include "vt.h"
 #define FLAGS 0x00000010
 
 #define THIS ((EnExRuppy*)thisx)
 
+#define DIVING_GAME ((EnDivingGame*)this->actor.attachedA)
+
 void EnExRuppy_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnExRuppy_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnExRuppy_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnExRuppy_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void EnExRuppy_DropIntoWater(EnExRuppy* this, GlobalContext* globalCtx);
+void EnExRuppy_BlowUp(EnExRuppy* this, GlobalContext* globalCtx);
+void EnExRuppy_Collect(EnExRuppy* this, GlobalContext* globalCtx);
```

Comment:
this one is touch cause this makes it sound like the rupee collects itself. As an action itd really be like Ruppy_Collected or maybe even a better word